templates/webapi/main/build_progressbar.html.ep

```diff
@@ -1,9 +1,10 @@
-% my $class = stash '';
+# Assign a default value to $class if it's not defined
+my $class = $stash->{class} // '';
```

Comment:
Have you tested this at all? I don't think this is working without preceding `% `. As-is, this code would just be rendered literally.